spec/service_consumers/pact_helper.rb

```diff
@@ -0,0 +1,78 @@
+require "pact/provider/rspec"
+require "webmock/rspec"
+require_relative "../support/organisations_api_pact_helper"
+require_relative "../support/error_responses"
+
+RSpec.configure do |config|
+  config.include ErrorResponses
+  config.around(:each) do |example|
+    respond_without_detailed_exceptions(&example)
+  end
+end
+
+Pact.configure do |config|
+  config.reports_dir = "spec/reports/pacts"
+  config.include WebMock::API
+  config.include WebMock::Matchers
+  config.include OrganisationsApiPactHelper
+end
+
+Pact.service_provider "Organisations API" do
+  honours_pact_with "GDS API Adapters" do
+    pact_uri "../gds-api-adapters/spec/pacts/gds_api_adapters-collections_organisation_api.json"
+  end
+end
+
+Pact.provider_states_for "GDS API Adapters" do
+  provider_state "there is a list of organisations" do
```

Comment:
Ah this is a mistake Ben sorry you spent time looking into it. The consumer test is matching child and superseded orgs with an `.each_like({})` which you are right, requires an array of one or more element. I simplified the matcher so it was happy with an empty array (because we don't always send those keys) but I omitted to push that commit! So it is passing for me locally, but would not have passed for you. I will amend the matcher in the consumer tests 👍